crates/store/re_types/definitions/rerun/blueprint/archetypes/dataframe_query.fbs

```diff
@@ -49,3 +50,38 @@ table DataframeQuery (
     time_range_queries: rerun.blueprint.components.TimeRangeQueries ("attr.rerun.component_optional", nullable,order: 500);
 }
 
+
+
+/// The query for the dataframe view.
+table DataframeQueryV2 (
+    "attr.rerun.scope": "blueprint"
+) {
+    // --- Optional ---
+
+    /// The timeline for this query.
+    ///
+    /// If unset, the time panel's timeline is used and stored.
+    timeline: rerun.blueprint.components.TimelineName ("attr.rerun.component_optional", nullable, order: 100);
+
+    /// If set, a range filter is applied.
+    ///
+    /// Note: will be unset as soon as `timeline` is changed.
+    //TODO(ab): should this use the list-with-timelines trick?
+    range_filter: rerun.blueprint.components.RangeFilter ("attr.rerun.component_optional", nullable, order: 200);
```

Comment:
TODO: change that to `filter_by_range`

Reply:
@jleibs the naming of these fields is probably something we should sync over—it would be very weird to have something different here and in the data API.